plugin/inc/dataintegrity_tests/cartridgeitems_invalid_customfields.php

```diff
@@ -8,9 +8,9 @@
           and (   coalesce(ci.ref, '') = ''
                or coalesce(ci.mercury_code_field, '') = ''
                or coalesce(ci.compatible_mercury_codes_field, '') = ''
-               or coalesce(ci.atc_field, 0) = 0 
-               or coalesce(ci.life_coefficient_field, 0) = 0 
-               or coalesce(ci.supported_types_field, 0) = 0)
+               or coalesce(ci.atc_field, '0') = '0' 
+               or coalesce(ci.life_coefficient_field, '0') = '0' 
+               or coalesce(ci.supported_types_field, '0') = '0')
```

Comment:
The atc and life_coefficient fields should be decimals or ints, the supported_types is varchar, so the check should also chanhe
```suggestion
               or coalesce(ci.atc_field, 0) = 0 
               or coalesce(ci.life_coefficient_field, 0) = 0
               or coalesce(ci.supported_types_field, '') = '')
```

Also: I also observed somewhere that you made a similar change (a made a comment), I don't know what happened with that.
The best would probably be to re-check the type of every customfield (to match with the type in v2)

Reply:
Fixed.